modules/govuk/manifests/app/config.pp

```diff
@@ -135,6 +132,13 @@
         value   => $sentry_dsn;
     }
 
+    if $::govuk_node_class !~ /^development$/ {
+      govuk::app::envvar { "${title}-GOVUK_APP_LOGROOT":
+        varname => "GOVUK_APP_LOGROOT",
```

Comment:
Think the linter is expecting this to be single-quoted.

Reply:
whoops! thanks